CMakeLists.txt

```diff
@@ -149,6 +149,10 @@ if (EVMJIT)
     add_subdirectory(evmjit)
 endif()
 
+if (HERA)
+    add_subdirectory(hera)
```

Comment:
Think we need to add a line here to make it build properly:
```
    set(hera_include_dir ${CMAKE_SOURCE_DIR})
```

This overrides the variable set by hera.